glusterd2/brick/validation.go

```diff
@@ -20,11 +20,44 @@ const (
 
 // InitChecks is a set of checks to be run on a brick
 type InitChecks struct {
-	IsInUse  bool
-	IsMount  bool
-	IsOnRoot bool
+	IsInUse          bool
+	IsMount          bool
+	IsOnRoot         bool
+	IsCreateBrickDir bool
```

Comment:
I'm sorry I didn't notice this earlier. This can be named better (grammatically) to `CreateBrickDir`

Reply:
done